packages/thirdweb/src/exports/storage.ts

```diff
@@ -10,3 +10,4 @@ export {
   resolveArweaveScheme,
   type ResolveArweaveSchemeOptions,
 } from "../utils/arweave.js";
+export type { FileOrBufferOrString } from "../storage/upload/types.js";
```

Comment:
are we *sure* we have to export this?

this is meant to handle things between frontends and backends consistently, however the dashboard should only be interacting with things with `Files` and maybe `string`s, so why do we need to export this for usage in dashboard?

Reply:
WIll revert this change and work on any issue leftover